src/main/kotlin/uk/gov/dluhc/printapi/jobs/InitialRetentionPeriodDataRemovalJob.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dluhc.printapi.jobs
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import uk.gov.dluhc.printapi.database.entity.SourceType.VOTER_CARD
+import uk.gov.dluhc.printapi.service.CertificateDataRetentionService
+
+@Component
+class InitialRetentionPeriodDataRemovalJob(
+    private val certificateDataRetentionService: CertificateDataRetentionService
+) {
+
+    @Scheduled(cron = "\${jobs.remove-vca-initial-retention-period-data.cron}")
+    @SchedulerLock(name = "\${jobs.remove-vca-initial-retention-period-data.name}")
+    fun removeVoterCardInitialRetentionPeriodData() {
+        certificateDataRetentionService.removeInitialRetentionPeriodData(sourceType = VOTER_CARD)
+    }
```

Comment:
We could perhaps introduce additional methods for other "Certificate/Document" types or other "source types", but I don't yet know what the retention policy is for temporary certs or AEDs. I don't think the first (initial) retention policy applies to temp certs, but it might to AEDs. It's all up in the air and we'll have to cross that bridge when we get to it.